### PR TITLE
Task 2 docs standards

### PR DESCRIPTION
**Pull request by jerkur22**

### CODE REVIEW

**Comment by jerkur22** [comment]
[Task 2: Create a Template for Documentation and Coding Standards](https://trello.com/c/VdTfU8Xp/3-task-2-create-a-template-for-documentation-and-coding-standards)

File: docs/STYLE_GUIDE.MD [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+      it('handles click events', () => {
+        // click events
+      });
+    });
+    ```
+
+## Back End
+
+More simple rules
```
Comment (arikamat) [inline docs/STYLE_GUIDE.MD:53]: Remove this line

Comment (arikamat) [inline docs/STYLE_GUIDE.MD]: Remove this line

File: docs/STYLE_GUIDE.MD [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- Use snake case for functions **`login_var`**
+- Snake case for variables
+- Snake case for file names
+- File names should be short and relevant for example **`user.py`** for the router file that has user related api 
+
+**Tests**
+- Any files used in tester files should be put in **`test_files`** folder and be prefaced with **`test_[filetype]_[num].[file extension]`**
+- Test files should be inside **`backend`** folder and should be prefaced with **`test_[folder]_[filename].py`**
+  - folder could be routers,utils,schema etc.
+  - Example testing routers/user.py should have following test file **`test_routers_user.py`**
```
Comment (arikamat) [inline docs/STYLE_GUIDE.MD]: Should we add documentation and linting requirements? 

i.e. 
run ``black backend`` and ``black tests`` before pr

**Review by arikamat — APPROVED** [review]
LGTM!

**Pull request merged**